solr/bin/solr

```diff
@@ -626,6 +626,7 @@ function jetty_port() {
 # run a Solr command-line tool using the SolrCLI class;
 # useful for doing cross-platform work from the command-line using Java
 function run_tool() {
+  export SOLR_PORT="$(choose_default_solr_port)"
```

Comment:
would `(choose_default_solr_port())` be possible to signal it's a function?   just a spitball...